packages/client/helpers/build.ts

```diff
@@ -28,18 +26,16 @@ const browserBuildConfig: BuildOptions = {
 }
 
 // we define the config for proxy
-const proxyBuildConfig: BuildOptions = {
-  name: 'proxy',
+const edgeRuntimeBuildConfig: BuildOptions = {
+  name: 'edge',
   entryPoints: ['src/runtime/index.ts'],
-  outfile: 'runtime/proxy',
+  outfile: 'runtime/edge',
   bundle: true,
   minify: true,
   legalComments: 'none',
   define: {
     // that helps us to tree-shake unused things out
```

Comment:
comment still valid although former properties were fully replaced? Or does `NODE_CLIENT` now fulfill this function?

Reply:
`NOT_PRISMA_DATA_PROXY=false` actually means `NODE_CLIENT-true`, but is easier to parse, as Serhii suggested.